api/pullrequests/pr.model.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+    login: String,
+    avatar_url: String,
+  });
+
+const diffCommentSchema = new Schema({
+    ghId: Number,
+    body: String,
+    created: Date,
+    file: String,
+    user: UserSchema
+});
+
+const PrSchema = new Schema({
+    url: String,
+    state: String,
+    name: String,
+    createdAt: Date,
+    id: Number,
+    diffComments: [diffCommentSchema],
+    user: UserSchema,
+    created: {
```

Comment:
I'm wondering if we want to create our own dates, or use the timestamp that Github Gives us? Trying to figure out which would be better.

Reply:
It would almost be the same though right?